fcm_django/models.py

```diff
@@ -266,9 +260,13 @@ def handle_topic_subscription(
 FCMDeviceManager = _FCMDeviceManager.from_queryset(FCMDeviceQuerySet)
 
 
-class AbstractFCMDevice(Device):
-    DEVICE_TYPES = (("ios", "ios"), ("android", "android"), ("web", "web"))
+class DeviceType(models.TextChoices):
```

Comment:
Juste noticed you support django 2.2 so this might not be possible yet

Reply:
No worries we can make this a major release and drop 2.2 django support